lib/src/model/localized_tag.dart

```diff
@@ -25,6 +25,9 @@ class LocalizedTag extends JsonObject {
   factory LocalizedTag.fromJson(Map<String, dynamic> json) =>
       _$LocalizedTagFromJson(json);
 
+  static Map<String, dynamic> objToJson(LocalizedTag? tag) =>
+      tag != null ? tag.toJson() : {};
+
   @override
   Map<String, dynamic> toJson() => _$LocalizedTagToJson(this);
```

Comment:
I'm no going to reopen our initial arguments regarding the use of `json_serializable` but needing both `static Map<String, dynamic> objToJson` and `Map<String, dynamic> toJson` is not among the best code I've ever read.

Reply:
Yeah, it does (generally) feel a bit awkward working with JSON conversion is Dart/Flutter